site/app/templates/GlobalHeader.twig

```diff
@@ -330,3 +379,195 @@
         </li>
     {% endif %}
 {% endmacro %}
+
+
+<script>
+    
+
+    var currentImageIndex = 0;
+    var hiddenImages = getHiddenImages();
+    var seenImages = []; 
+    var images = document.getElementsByClassName('club-banners');
+    bannerElement = document.getElementById("banner");
+    bannerElement.style.display = 'none';
+    bannerElement.style.width = "1%";
+
+
+
+    var currentImages = [];
```

Comment:
Use `let` or `const` instead of `var` whenever possible. Our linting rules don't hit the twig files so this all gets checked manually.